cmd/backup.go

```diff
@@ -319,3 +320,84 @@ func newTableBackupCommand() *cobra.Command {
 	}
 	return command
 }
+
+// newRawBackupCommand return a raw kv range backup subcommand.
+func newRawBackupCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "raw",
+		Short: "backup a raw kv range",
+		RunE: func(command *cobra.Command, _ []string) error {
+			ctx, cancel := context.WithCancel(defaultContext)
+			defer cancel()
+
+			mgr, err := GetDefaultMgr()
+			if err != nil {
+				return err
+			}
+			defer mgr.Close()
+
+			client, err := backup.NewBackupClient(ctx, mgr)
+			if err != nil {
+				return err
+			}
+			u, err := storage.ParseBackendFromFlags(command.Flags(), FlagStorage)
+			if err != nil {
+				return err
+			}
+
+			err = client.SetStorage(u)
+			if err != nil {
+				return err
+			}
+
+			start, err := command.Flags().GetString("start")
+			if err != nil {
+				return err
+			}
+
+			end, err := command.Flags().GetString("end")
+			if err != nil {
+				return err
+			}
+
+			rate, err := command.Flags().GetUint64("ratelimit")
+			if err != nil {
+				return err
+			}
+			concurrency, err := command.Flags().GetUint32("concurrency")
+			if err != nil {
+				return err
+			}
+			if concurrency == 0 {
+				return errors.New("at least one thread required")
+			}
+
+			ranges := []backup.Range{{StartKey: []byte(start), EndKey: []byte(end)}}
```

Comment:
Oh, looks like it's not proper to parse keys like this.

Reply:
yes, add function [`ParseKey`](https://github.com/pingcap/br/pull/101/files#diff-83d7917e5fba66e42677eb2d93c47ed5R15), it can parse hex string

Reply:
Please also consider check the correctness of user's input, return an error if endKey < startKey.